### PR TITLE
Added ability to remove NonEmpty check from password field in login form

### PR DESCRIPTION
**Pull request by Alroniks**
Related to https://oat-sa.atlassian.net/browse/TAO-3139

### CODE REVIEW

File: actions/form/class.Login.php [hunk 1]
```diff
@@ -19,79 +19,56 @@
...
+
+        $loginElt = tao_helpers_form_FormFactory::getElement('login', 'Textbox');
+        $loginElt->setDescription(__("Login"));
+        $loginElt->setAttributes(['autofocus' => 'autofocus']);
+        $loginElt->addValidator(tao_helpers_form_FormFactory::getValidator('NotEmpty'));
+        $this->form->addElement($loginElt);
+
+        $passElt = tao_helpers_form_FormFactory::getElement('password', 'Hiddenbox');
+        $passElt->setDescription(__("Password"));
+        if (!empty($this->data['emptyPassword'])) {
```
Comment (antoinerobin) [inline actions/form/class.Login.php]: this field is never set, you didn't pass the params to the constructor

**Comment by Alroniks** [comment]
@antoinerobin Fixed issue and version

**Comment by Alroniks** [comment]
@antoinerobin After discussion with a customer this feature not needed more for them so I can close it but maybe it may need for us in the future. What do you think?

**Comment by antoinerobin** [comment]
I think you can close it, I don't see a use case for an empty password

**Comment by Alroniks** [comment]
No need more

**Pull request closed without merging**